--- setup.py	2015-03-02 13:53:02.978523562 -0800
+++ patch-files/setup.py	2015-03-02 13:53:18.589537865 -0800
@@ -143,6 +143,6 @@
   package_data = package_data,
   ext_modules = [Extension('h5py.x',['x.c'])],  # To trick build into running build_ext
   requires = ['numpy (>=1.6.1)', 'Cython (>=0.17)'],
-  install_requires = ['numpy>=1.6.1', 'Cython>=0.17'],
+  install_requires = [],
   cmdclass = CMDCLASS,
 )
